docs/source/operators/config-availability.md

```diff
@@ -0,0 +1,106 @@
+# Availability modes
+
+Enterprise Gateway can be optionally configured in one of two "availability modes": _active-active_ and _active-passive_. Both modes require that kernel session persistence also be enabled via `KernelSessionManager.enable_persistence=True`.
+
+```{warning}
+**Availability modes and kernel session persistence should be considered experimental!**
+Known issues include:
+1. Culling configurations do not account for different nodes and therefore could result in the premature culling of kernels.
+2. Each "node switch" requires a manual reconnect to the kernel.
+
```

Comment:
Are  the above issues only with "active-active" mode and not with "active-passive" mode of EG?

Reply:
I think reconnecting is necessary for both forms.  Even with "active-active", because we still expect/advise affinity with the managed kernel, you shouldn't run into an issue where the original EG is still running and can cull the kernel pre-maturely.  Only if the affinity is not configured could the kernel be culled prematurely from the previous node.  Since this is a general warning callout, I'm not sure we need to clarify, but we should probably take a closer look at this.